fe/fe-core/src/main/java/com/starrocks/sql/optimizer/base/ColumnRefFactory.java

```diff
@@ -93,6 +93,17 @@ public void updateColumnRefToColumns(ColumnRefOperator columnRef, Column column,
         columnRefToTable.put(columnRef, table);
     }
 
+    /*
+    public void updateColumnRef(ColumnRefOperator oldColumnRef, ColumnRefOperator columnRef) {
+        Column column = columnRefToColumns.remove(oldColumnRef);
+        Table table = columnRefToTable.remove(oldColumnRef);
```

Comment:
Is this function necessary?

Reply:
I forget to remove it. I will remove it.